pages/portfolio.html

```diff
@@ -4,9 +4,30 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="/style.css" rel="stylesheet">
   <title>Document</title>
 </head>
 <body>
-  
+  <header>
+    <h1>
+      Christina Webber
+    </h1>
+    <h2>
+      a developer in development
+    </h2>
+  </header>
+  <main>
+    <section class="container">
+      <nav class="navigation">
+        <div class="first"><a href="/pages/index.html">main</a></div>
+        <div class="second"><a href="/pages/about.html">about</a></div>
+      </nav>
+      <div class="pr1"><a href="https://github.com/crw-b/viewing-party">Viewing Party</a></div>
+      <div class="pr2"><a href="https://github.com/crw-b/task-list-api">Task List API</a></div>
+      <div class="pr3"><a href="https://github.com/crw-b/swap-meet">Swap Meet</a></div>
+      <div class="pr4"><a href="https://github.com/crw-b/solar-system-api">Solar System API</a></div>
```

Comment:
These are logically a list of projects, so it would be nice if they were presented using list tags, though this would require some other changes to the layout rules to maintain the current way the projects are shown.